konduit-serving-common-tests/pom.xml

```diff
@@ -58,5 +58,38 @@
             <artifactId>jetty-server</artifactId>
             <version>9.4.19.v20190610</version>
         </dependency>
+        <dependency>
+            <groupId>org.reflections</groupId>
+            <artifactId>reflections</artifactId>
+            <version>0.9.9</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>2.4.8</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
This groovy dependency should not be necessary.
Also vertx-core should not be necessary.

Also remove `<scope>compile</scope>` from these dependencies that remain (compile scope is the default so specifying it as such is unnecessary)